R/modify_abbreviation.R

```diff
@@ -49,13 +49,21 @@ modify_abbreviation <- function(x, abbreviation, text_interpret = c("md", "html"
 
 #' @export
 #' @rdname modify_abbreviation
-remove_abbreviation <- function(x, abbreviation) {
+remove_abbreviation <- function(x, abbreviation = NULL) {
   set_cli_abort_call()
   updated_call_list <- c(x$call_list, list(modify_footnote_body = match.call()))
 
   # check inputs ---------------------------------------------------------------
   check_class(x, "gtsummary")
-  check_string(abbreviation)
+  check_string(abbreviation, allow_empty = TRUE)
+
+  # remove all abbreviations if abbreviation=NULL ------------------------------
```

Comment:
I would add a line documenting this behavior

Reply:
ah yes! Very good point, thank you!!

Reply:
added!